.travis/smoke-test.sh

```diff
@@ -3,11 +3,14 @@ set -ex
 
 COMPOSE_PROJECT_NAME="${COMPOSE_PROJECT_NAME:-sampleconfiguration}"
 
+echo "Wait for containers health"
 .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_dummy-api_fpm_1"
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_browser_fpm_1"
+.scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_web_1"
```

Comment:
Makes sense to wait on multiple containers in that script now.

Reply:
Will make a new backward-compatible version, not sure if there are other arguments but not for now.